src/linear_algebra/basis.lean

```diff
@@ -419,7 +420,7 @@ b.reindex_range_repr' _ rfl
 
 section fintype
 
-variables [fintype ι]
+variables [fintype ι] [decidable_eq M]
```

Comment:
This is needed since it is an argument to `finset.univ.image b`, which appears in the type